addon/components/pix-link-back.hbs

```diff
@@ -1,13 +1,35 @@
-<LinkTo @route={{this.route}}
-    @activeClass="pix-link-back--active"
-    @model={{this.model}}
-    class="pix-link-back pix-link-back--{{this.shade}}"
-    ...attributes>
+{{#if @model}}
```

Comment:
```suggestion
{{#if this.model}}
```
?

Reply:
Au final pas besoin de passer par le controller avec `this.model`, on peut faire #if @model avec @model venant directement de là où est appelé PixLinkBack